packages/agreement-process/src/resources/templates/documents/agreementContractTemplate.html

```diff
@@ -55,10 +55,16 @@ <h1>Richiesta di fruizione</h1>
       <strong>{{eserviceId}}</strong>, versione {{descriptorVersion}},
       contraddistinta da id <strong>{{descriptorId}}</strong> (di seguito
       “e-service”), reso disponibile dall’aderente
-      <strong>{{producerName}}</strong> {{#if producerIpaCode}}(codice IPA:
+      {{#if delegationId}}
+      <strong>{{delegatorName}}</strong> {{#if delegatorIpaCode}}(codice IPA:
+      <strong>{{delegatorIpaCode}}</strong> - di seguito “erogatore”){{else}}(di seguito “erogatore”), per tramite dell’aderente{{/if}}
```

Comment:
", per tramite dell’aderente" shall be outside the if

```suggestion
      <strong>{{delegatorName}}</strong>{{#if delegatorIpaCode}} (codice IPA:
      <strong>{{delegatorIpaCode}}</strong> - di seguito “erogatore”){{else}}
      (di seguito “erogatore”){{/if}}, per tramite dell’aderente
```

Reply:
Solved [here](https://github.com/pagopa/interop-be-monorepo/pull/1175/commits/55cce150aea9db2be125a05e939801e8e2f9c215)